SWLOR.Game.Server/Feature/AIDefinition/GenericAIDefinition.cs

```diff
@@ -32,7 +32,106 @@ static float CalculateAverageHP(uint creature)
             var lowestHPAllyRace = GetRacialType(lowestHPAlly);
             var allyCount = allies.Count;
             var activeConcentration = Ability.GetActiveConcentration(self).Feat;
-            
+
+            // Benevolence
+            if (CheckIfCanUseFeat(self, target, FeatType.Benevolence1, () => allyHPPercentage < 100))
```

Comment:
Is it overkill to target if there's even 1 HP lost?

Reply:
Based on my testing, there's such a long delay between when it starts taking damage and when it would use healing that this the best way to ensure they keep themselves moderately healed.